script.js

```diff
@@ -1,27 +1,47 @@
 const BurgerEl = document.getElementById("burger");
-const NavItemsEl = document.getElementById("navItems")
+const NavItemsEl = document.getElementById("navItems");
+const menuOpen = document.createElement("div");
+menuOpen.classList.add("menuOpen");
+const overlay = document.querySelector('#overlay');
+overlay.appendChild(menuOpen);
 
-function burgerToggle(){
-    if(NavItemsEl.style.visibility == "hidden"){
+function burgerToggle() {
+    if (NavItemsEl.style.visibility == "hidden") {
         NavItemsEl.style.visibility = "visible";
     }
-    else{
+    else {
         NavItemsEl.style.visibility = "hidden";
-    } 
+    }
 }
 
-BurgerEl.addEventListener("click", ()=>{    
-    NavItemsEl.classList.toggle("active")   
+BurgerEl.addEventListener("click", () => {
+    console.log('click open');
```

Comment:
console.log's should probably be removed before deployment :) Add a `// TODO: remove` here :)